app/src/main/java/com/kms/cura/utils/DataUtils.java

```diff
@@ -113,4 +126,16 @@ public static void loadProfile(UserEntity user, ImageView profile, Context conte
         }
     }
 
+    public static List<AppointmentEntity> getDoctorAppt(List<AppointmentEntity> appts){
```

Comment:
But exclude AppointmentEntity.PENDING_STT and AppointmentEntity.PATIENT_CANCEL_STT
 and AppointmentEntity.REJECT_STT right ? Should make it clear.